src/actions/index.js

```diff
@@ -4,26 +4,33 @@ import {
   FETCH_POST,
   FETCH_MEETUP,
   LOGIN,
-  REGISTER
+  REGISTER,
+  VIEW_MEETUP
 } from "./types";
 import axios from "axios";
 import Swal from "sweetalert2";
+import { dispatch } from "rxjs/internal/observable/pairs";
+
+
 
 const apiUrl = "https://bencyn-questioner.herokuapp.com/api/v2";
 
+//  View Meetups Action Logic
+// const alert = useAlert()
 export const login = (data, history) => {
   return dispatch => {
     return axios
       .post(apiUrl + "/auth/login", { ...data })
```

Comment:
You can put this in a helper function and have it in a helper file in that you can reuse with all the actions. This helper function will make API calls therefor you can pass parameters including `method`, `endpoint`, `payload/data`. This will improve your code quality. Let me know if you need more insight on the same.